Pipfile.lock

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I was checking https://github.com/pypa/pipfile and it appears that you do not need to supply this file as it should be generated.  If it's generated, then this shouldn't be included in the repo.  Is there another reason why the lock file is included?